app/resolve/cli/__init__.py

```diff
@@ -20,7 +25,7 @@ def save_to_output(result: RquestResult, destination: str) -> None:
     """
     if not destination.endswith(".json"):
```

Comment:
this is the legacy code but yes, try--except block will catch it, but I think it would provide clearer details about this specific error when it happens